install/rpm/nginx/nginx.conf

```diff
@@ -14,114 +15,101 @@ events {
-	default_type                  application/octet-stream;
+	include                         /etc/nginx/mime.types;
+	default_type                    application/octet-stream;
 	# Compression
-	gzip                          on;
-	gzip_static                   on;
-	gzip_vary                     on;
-	gzip_comp_level               6;
-	gzip_min_length               1024;
-	gzip_buffers                  16 8k;
-	gzip_http_version             1.1;
-	gzip_types                    text/plain text/css text/javascript text/js text/xml
-	  application/json application/javascript application/x-javascript application/xml
-	  application/xml+rss application/x-font-ttf image/svg+xml font/opentype;
-	gzip_proxied                  any;
-	gzip_disable                  "MSIE [1-6]\.";
-	# Cloudflare https://www.cloudflare.com/ips
-	set_real_ip_from              103.21.244.0/22;
-	set_real_ip_from              103.22.200.0/22;
-	set_real_ip_from              103.31.4.0/22;
-	set_real_ip_from              104.16.0.0/12;
-	set_real_ip_from              108.162.192.0/18;
-	set_real_ip_from              131.0.72.0/22;
-	set_real_ip_from              141.101.64.0/18;
-	set_real_ip_from              162.158.0.0/15;
-	set_real_ip_from              172.64.0.0/13;
-	set_real_ip_from              173.245.48.0/20;
-	set_real_ip_from              188.114.96.0/20;
-	set_real_ip_from              190.93.240.0/20;
-	set_real_ip_from              197.234.240.0/22;
-	set_real_ip_from              198.41.128.0/17;
-	#set_real_ip_from  2400:cb00::/32;
-	#set_real_ip_from  2405:b500::/32;
-	#set_real_ip_from  2606:4700::/32;
-	#set_real_ip_from  2803:f800::/32;
-	#set_real_ip_from  2c0f:f248::/32;
-	#set_real_ip_from  2a06:98c0::/29;
-	real_ip_header                CF-Connecting-IP;
+	gzip                            on;
+	gzip_static                     on;
+	gzip_vary                       on;
+	gzip_comp_level                 6;
+	gzip_min_length                 1024;
+	gzip_buffers                    128 4k;
+	gzip_http_version               1.1;
+	gzip_types                      text/css text/javascript text/js text/plain text/richtext text/shtml text/x-component text/x-java-source text/x-markdown text/x-script text/xml image/bmp image/svg+xml image/vnd.microsoft.icon image/x-icon font/otf font/ttf font/x-woff multipart/bag multipart/mixed application/eot application/font application/font-sfnt application/font-woff application/javascript application/javascript-binast application/json application/ld+json application/manifest+json application/opentype application/otf application/rss+xml application/ttf application/truetype application/vnd.api+json application/vnd.ms-fontobject application/wasm application/xhtml+xml application/xml application/xml+rss application/x-httpd-cgi application/x-javascript application/x-opentype application/x-otf application/x-perl application/x-protobuf application/x-ttf;
+	gzip_proxied                    any;
+	gzip_disable                    "msie6";
```

Comment:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=867024

Maybe it is time to just remove it... 

Microsoft IE 6 has been dead for